helm/formatting_guidelines.md

```diff
@@ -0,0 +1,54 @@
+# Helm Chart Formatting Guidelines
+
+## Templating
+
+When using helm templating calls we include double spaces inside the double brackets
+like this:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-configmap
```

Comment:
I think this single line would be enough, you only do a single line in the comparison aswell.

Reply:
I think It's better so see within some context. But added the same example in the comparison for completeness :)